### PR TITLE
UI in response of POST /bot

### PR DESCRIPTION
**Pull request by kriscfoster**
@kamilsmuga @tausifmuzaffar an object containing the name & size of UI zip is in response.. Is this overkill? If so which would be better to have?

### CODE REVIEW

File: api/v1/controllers/bots.js [hunk 1]
```diff
@@ -86,12 +86,15 @@ module.exports = {
     const resultObj = { reqStartTime: new Date() };
     const reqObj = req.swagger.params;
     const seqObj = {};
+    const uiObj = {};
     try {
       for (const param in reqObj) {
         if (reqObj[param].value) {
           if (typeof (reqObj[param].value) === 'object' &&
```
Comment (shriramshankar) [inline api/v1/controllers/bots.js]: I think swagger will take care of this validation.

File: api/v1/controllers/bots.js [hunk 2]
```diff
@@ -86,12 +86,15 @@ module.exports = {
     const resultObj = { reqStartTime: new Date() };
     const reqObj = req.swagger.params;
     const seqObj = {};
+    const uiObj = {};
```
Comment (iamigo) [inline api/v1/controllers/bots.js]: no tests?

File: api/v1/controllers/bots.js (hunk 1 above)
Comment (kriscfoster) [inline api/v1/controllers/bots.js]: @shriramshankar I'm making that object to overwrite the value for UI in the response which is long binary data. I'm overwriting it at line 107.

File: api/v1/controllers/bots.js (hunk 2 above)
Comment (kriscfoster) [inline api/v1/controllers/bots.js]: @iamigo I added expects for the UI in response now in api POST & PUT

**Pull request merged**